packages/amplify-app/src/test-helpers/amplify-app-cleanup.js

```diff
@@ -0,0 +1,40 @@
+const rimraf = require('rimraf');
+const nexpect = require('nexpect');
+const path = require('path');
+
+function getCLIPath() {
+  return path.join(__dirname, '..', '..', '..', 'amplify-cli', 'bin', 'amplify');
+}
+
+function amplifyDelete() {
+  return new Promise((resolve, reject) => {
+    nexpect
+      .spawn(getCLIPath(), ['delete'])
+      .wait('Are you sure you want to continue?')
+      .sendline('\r')
+      .sendline('')
+      .wait('Project deleted locally.')
+      .run(function(err) {
+        if (!err) {
+          resolve();
+        } else {
+          reject(err);
+        }
+      });
+  });
+}
+
+function deleteAmplifyAppFiles() {
```

Comment:
can't we delete a full folder here in 1 step? rimraf support globbing as well.

Reply:
Yeah this is unnecessary with test folders being removed in the test suites. Removed